### ISSUE TITLE
Statistics Line Chart: How to set your own line colors?

### ISSUE THREAD

**Issue by geoffsari** [crocodic-studio/crudbooster#1466]
Hi there, I have managed to get my statistics line chart displaying correctly using the following **SQL Query Line**:

`select date_format(date_reported,"%Y-wk%w") as label, format(AVG(price),2) as value from price_summary where product_code = "RR2001L" and year(date_reported) = year(CURDATE()) group by label Order by label`

However, I wish to define my own line colors as each of my products have brand coloring, so I tried adding `"#F82808" as lineColors` as an alias in my sql query, hoping to get the line color to change, but it does not change the line color as intended. However, the line still displays with the predefined colors.

`select date_format(date_reported,"%Y-wk%w") as label, format(AVG(price),2) as value, "#F82808" as lineColors from price_summary where product_code = "RR2001L" and year(date_reported) = year(CURDATE()) group by label Order by label`

So is there are way to do this with the internal statistics builder or should I create my own custom line chart for this? 

Thanks

My Specs:
PHP 7.1.30
Laravel Version : 5.7.28
CrudBooster Version : 5.4.15
Windows 10 OS

**Comment by geoffsari** [comment]
To benefit those who may face a similar issue. I could not change line colors with Crudbooster's statistic builder for Line Chart widget. So ended up building my own custom dashboards instead. 

I used this package https://github.com/fxcosta/laravel-chartjs along with this https://github.com/imanghafoori1/laravel-widgetize, which is also part of CRUDBooster dependency package, so did not have to pull that in. 

Basically using this two packages, you can create your own chart and statistics box widgets, so you reuse this widget anywhere within your project.
